astropy/coordinates/tests/test_frames.py

```diff
@@ -1178,16 +1183,14 @@ def test_galactocentric_defaults():
         else:
             assert getattr(galcen_40, k) == getattr(galcen_latest, k)
 
-    reset_galactocentric_defaults()
-
 
-def test_galactocentric_default_warning():
+def test_galactocentric_default_warning(reset_galactocentric_defaults):
     from astropy.coordinates import (Galactocentric,
                                      galactocentric_frame_defaults)
 
     # Note: this is necessary because the doctests may alter the state of the
     # class globally and simultaneously when the tests are run in parallel
-    reset_galactocentric_defaults()
+    # reset_galactocentric_defaults()
```

Comment:
Oops, missed in review...

Reply:
So is this line supposed to be removed?

Reply:
Indeed, no reason to just comment it out, it is now done in `setup` and `teardown`. Indeed, the comment above can go too...

Reply:
OK, I'll fold it into #7928.